provision/trigger.go

```diff
@@ -0,0 +1,76 @@
+package provision
+
+import (
+	"fmt"
+	"scaling_manager/cluster"
+)
+
+// Input:
+// Description:
+//
+//	getRecommendation will fetch the recommendation from recommendation queue and clear the queue.
+//	It will populate the command queue which contains all the details to scale out the cluster.
+//
+// Return:
+//
+//	getRecommendation will return the command struct which will contains all the details to scale out the cluster.
+func getRecommendation() Command {
+	var command Command
+	// Fetch the recommendation from the queue
```

Comment:
Before clearing the queue check the state which should be normal and the cluster state which should be green and if criteria match then only clear the queue.